libbeat/docs/communitybeats.asciidoc

```diff
@@ -11,7 +11,7 @@ https://github.com/awormuth/amazonbeat[amazonbeat]:: Reads data from a specified
 https://github.com/radoondas/apachebeat[apachebeat]:: Reads status from Apache HTTPD server-status.
 https://github.com/goomzee/burrowbeat[burrowbeat]:: Monitors Kafka consumer lag using Burrow.
 https://github.com/goomzee/cassandrabeat[cassandrabeat]:: Uses Cassandra's nodetool cfstats utility to monitor Cassandra database nodes and lag.
-https://github.com/hartfordfive/cloudflarebeat[cloudflarebeat]:: Index log entries from the Cloudlflare Enterprise Log Share API
+https://github.com/hartfordfive/cloudflarebeat[cloudflarebeat]:: Indexes log entries from the CloudFlare Enterprise Log Share API.
```

Comment:
You're right. I searched on the API name and found a page that used the name incorrectly. I'll fix the PRs and resubmit. Thanks!